source/reference/mongosync.txt

```diff
@@ -32,6 +32,8 @@ line options.
    - Standalone MongoDB instances are not supported. :ref:`Convert the
      standalone instance <server-replica-set-deploy-convert>` to a
      replica set before using {+c2c-product-name+}. 
+   - ``mongosync`` does not support Atlas shared clusters or Serverless instances. For more information, see :ref:`c2c-limitations`. 
```

Comment:
[nit] Sorry I didn't call this out the first time around, but for product names like Atlas, we typically want to use the snooty constant! Can see the whole list of them on the snooty.toml file.

Also, might want to double check with eng/product, but I think "serverless" shouldn't be capitalized. 

```suggestion
   - ``mongosync`` does not support {+atlas+} shared clusters or serverless instances. For more information, see :ref:`c2c-limitations`. 
```

Reply:
Thank you @ajhuh-mdb !